server/controllers/conversation.js

```diff
@@ -0,0 +1,67 @@
+const Conversation = require("../models/Conversation");
+const asyncHandler = require("express-async-handler");
+
+// @route POST /conversations
+// @desc Create new conversation
+// @access Public
+exports.createConversation = asyncHandler(async (req, res, next) => {
+  const { dogOwnerId, dogSitterId } = req.body;
+
+  try {
```

Comment:
We don't need a try/catch block as the `asyncHandler` handles promise rejections and forwards them to the error middleware

Ditto throughout!